lib/Models/CatalogItem.js

```diff
@@ -373,6 +389,23 @@ CatalogItem.defaultSerializers.attribution = function(catalogItem, json, propert
     }
 };
 
+CatalogItem.defaultSerializers.url = function(catalogItem, json, prototypeName) {
```

Comment:
These three serializers shouldn't be necessary, because the default serialization will do the same thing.

However, it'd be useful to avoid serializing `dataUrl` when it is identical to `url`.